src/onboarding/github.md

```diff
@@ -0,0 +1,61 @@
+# GitHub Tickets
+
+GitHub is a service that provides code hosting and versioning. We also use it for its collaborative features, such as issue/bug tracking and project boards. This documents explains how to use GitHub to keep track of tasks.
+
+## Repositories
+
+Here is the homepage for the [toronto-community-network](https://github.com/tomeshnet/toronto-community-network) repository.
+
+![homepage](./images/homepage.png)
+
+A repository (or "repo") is like a project on GitHub, with a collection of files and a name. You can see the name of this one, `toronto-community-network` and that it was created by the [tomeshnet](https://github.com/tomeshnet) organization.
+
+You can see an overview of the files and folders in the middle of screen, and below that you can see what's called the README, which is a file that explains what the repository is for. If you click on `README.md` you'll see it has the same contents as the text on the homepage.
```

Comment:
Again not sure about the tone

"You can see an overview of the files and folders" vs "An overview of the files and folders is displayed in the middle of the screen"